### DISCUSSION TITLE
Netbox

### DISCUSSION THREAD

**Discussion by 33Fraise33** [selfhst/icons#118] [New Icon Request]
Hello,

Netbox has a new logo but currently the square is not hosted anywhere. 
Logos can be found in the docs: https://github.com/netbox-community/netbox/tree/develop/docs (netbox_logo_dark.svg, netbox_logo_light.svg)

**Comment by shollyethan** [comment]
NetBox has been added - thanks!

**Reply by 33Fraise33** [reply]
Thanks, that was rather swift ;)
